src/cpp/server/server_builder.cc

```diff
@@ -217,8 +216,8 @@ std::unique_ptr<Server> ServerBuilder::BuildAndStart() {
   // ServerBuilder's AddCompletionQueue() method (those completion queues
   // are in 'cqs_' member variable of ServerBuilder object)
   std::shared_ptr<std::vector<std::unique_ptr<ServerCompletionQueue>>>
-      sync_server_cqs(
-          new std::vector<std::unique_ptr<ServerCompletionQueue>>());
+      sync_server_cqs = std::make_shared<
```

Comment:
You can say `sync_server_cqs(std::make_shared< ...`

that is, you can use it as a constructor argument for clarity and consistency